### PR TITLE
Updated and Improved LICENSE

### PR DESCRIPTION
**Pull request by Sumit1777**
Update and Improved LICENSE from (C) to ©

### CODE REVIEW

**Review by NitinNair89 — COMMENTED** [review]
This looks like it is done solely for Hacktoberfest 2020.

@3x, Please close this PR and add "invalid" label :slightly_smiling_face:

**Review by NitinNair89 — COMMENTED** [review]
This looks like it is done solely for Hacktoberfest 2020.

@3x, Please close this PR and add "invalid" label :slightly_smiling_face:

**Comment by Sumit1777** [comment]
yes, you Find that it is absolutely done for hactoberfest, but I haven't
done any irrelevant change and I doesn't making any spamming.

**Comment by NitinNair89** [comment]
@Sumit1777 I'm not a maintainer of this repo. But, as a contributor myself I don't think adding the copyright symbol improves the LICENSE document. And that's why I commented.

Check with the maintainer about how you can actually make some valuable contributions for this repo.

Peace. :slightly_smiling_face:

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Example (https://github.com/3x)
+Copyright © 2020 Example (https://github.com/3x)
```
Comment (shivasurya) [inline LICENSE:3]: 🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️

**Pull request closed without merging**